python/Dockerfile

```diff
@@ -59,7 +59,6 @@ WORKDIR /connector
 COPY --from=builder /builder/connector ./$CONNECTOR_NAME
 COPY --from=builder /builder/python ./python
 
-ENV PYTHONPATH="/connector/$CONNECTOR_NAME"
```

Comment:
This is the change that broke things -- all python packages (except `source-hubspot`) are now complaining that 

```
File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/connector/source-airtable/__main__.py", line 2, in <module>
    from source_airtable import SourceAirtable
ImportError: cannot import name 'SourceAirtable' from 'source_airtable' (unknown location)
```